packages/ckeditor5-clipboard/docs/features/paste-plaintext.md

```diff
@@ -0,0 +1,45 @@
+---
+menu-title: Paste plain text
+category: features-pasting
+order: 30
+---
+
+{@snippet features/build-paste-source}
+
+# Pasting plain text
+
+The plain text pasting feature is provided through the {@link module:clipboard/clipboard~Clipboard} plugin, which is responsible for the integration with the native clipboard and intercepts all native events like `copy`, `cut` or `drop` and handles them on its side. The goal is to not allow the browser to touch the content in the rich text editor which would lead to the browser messing it up.
```

Comment:
I'm not sure about the last sentence, what is it supposed to mean?

Reply:
It's taken straight from the Clipboard documentation. It means the editor takes over the clipboard handling, otherwise the browser may produce unexpected results.

Reply:
Reworded.